### PR TITLE
docs: worlds updates

### PR DESCRIPTION
**Pull request by pentreathm**

### CODE REVIEW

File: content/creator/scenes/publishing/deploy-third-party.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ type: Document
 url: /creator/development-guide/deploy-third-party
 ---
 
-If you don't own parcels in Decentraland or are not ready to [deploy]({{< ref "/content/creator/scenes/publishing/publishing.md#the-test-server" >}}) your scene to Decentraland, you can upload your scene preview to run as an app in a free server.
+If you do not possess any parcels in Decentraland or a Decentraland [NAME](https://builder.decentraland.zone/names) to publish your scene to a [Worlds]({{< ref "/content/creator/worlds/about.md" >}}), or if you are not yet prepared to [deploy]({{< ref "/content/creator/scenes/publishing/publishing.md#the-test-server" >}}) your scene to Decentraland, there is an alternative option available. You can upload a preview of your scene to run as an application on a free server.
```
Comment (marianogoldman) [inline content/creator/scenes/publishing/deploy-third-party.md]:
```suggestion
If you do not possess any parcels in Decentraland or a Decentraland [NAME](https://builder.decentraland.zone/names) to publish your scene to a [World]({{< ref "/content/creator/worlds/about.md" >}}), or if you are not yet prepared to [deploy]({{< ref "/content/creator/scenes/publishing/publishing.md#the-test-server" >}}) your scene to Decentraland, there is an alternative option available. You can upload a preview of your scene to run as an application on a free server.
```

Comment (marianogoldman) [inline content/creator/scenes/publishing/deploy-third-party.md]: I think Heroku is no longer free.. so I would change this "run as an application on a free server" to "run as an application on a Heroku server"

File: content/creator/scenes/publishing/publishing.md [hunk 2]
```diff
@@ -54,7 +56,11 @@ Open your scene's _scene.json_ file and complete the following data:
 
 ### Via the Decentraland Editor
 
-Make sure you've [installed the Decentraland editor]({{< ref "/content/creator/scenes/getting-started/installation-guide.md#the-decentraland-editor" >}}).
+Make sure you've [installed the Decentraland editor]({{< ref "/content/creator/scenes/getting-started/installation-guide.md#the-decentraland-editor" >}}). 
+
+{{< hint warning >}}
+**📔 Note**:  Deployment to Worlds is not yet supported from the Editor.
```
Comment (marianogoldman) [inline content/creator/scenes/publishing/publishing.md:62]: Is this not working already?

**Review by marianogoldman — APPROVED** [review]

File: content/creator/scenes/publishing/publishing.md (hunk 2 above)
Comment (pentreathm) [reply content/creator/scenes/publishing/publishing.md:62]: no, I did a test and it's not possible,

Comment (nearnshaw) [reply content/creator/scenes/publishing/publishing.md:62]: I believe it is already working on the editor, yes.
At least there´s a "Publish scene to your world" option in the UI if you click the three dots for more options

**Pull request merged**